conda/recipes/pynvjitlink/recipe.yaml

```diff
@@ -0,0 +1,71 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.
+schema_version: 1
+
+context:
+  cuda_version: ${{ (env.get("RAPIDS_CUDA_VERSION") | split("."))[:2] | join(".") }}
```

Comment:
This project does things a bit differently than the rest of RAPIDS by using the `CUDA_VERSION` file noted above. Let's see if we can align on that somehow. Perhaps that can be done via setting an environment variable above